src/types/actions.lua

```diff
@@ -3,13 +3,13 @@ export type Action<Type = any> = {
 	type: Type,
 }
 
-export type AnyAction = Action & {
+export type AnyAction = {
 	[string]: any,
-}
+} & Action
```

Comment:
Does this make a difference semantically or is this just preference for readability?

Reply:
It shouldn't make a difference semantically, but it does for... reasons.

Reply:
Or maybe more accurately, the order coerces the type engine to evaluate this type more closely to its intended purpose.